packages/dcos-check-runner/extra/dcos-checks-poststart.service

```diff
@@ -1,9 +1,15 @@
 [Unit]
 Description=DC/OS Poststart Checks: Run node-poststart checks
+
 [Service]
+Restart=always
+RestartSec=5
+StartLimitInterval=0
```

Comment:
Do we know how these settings interact with the corresponding timer settings?

```
[Unit]
Description=DC/OS Checks Timer: timer for DC/OS Checks service
[Timer]
OnCalendar=minutely
AccuracySec=10
```

Reply:
We don't want any of these settings. This unit doesn't provide a long-running service, so there's never a need to restart.